### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by galeksandrp**
Added

- rsdn.ru
- www.tldrlegal.com
- (www.|ipv6.)lurkmore.*
- (www.|ipv6.)nnm(-)club.(me|to)
- (www.)flibusta.is

Deleted

- www.modern.ie
- *.tldrlegal.com

### CODE REVIEW

File: src/chrome/content/rules/Hstor.org.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	<securecookie host="^hstore\.org$" name=".+" />
 
 
-	<rule from="^http:"
-		to="https:" />
+	<rule from="http://(?:beta\.)?hstor.org/"
```
Comment (fuglede) [inline src/chrome/content/rules/Hstor.org.xml]: The <code>.</code> should be a <code>\\.</code>.

**Comment by J0WI** [comment]
This branch conflicts with some other changes.

**Comment by davidabian** [comment]
Note that this build started two hours ago. It seems to be frozen, and [is blocking all the other builds.](https://travis-ci.org/EFForg/https-everywhere/pull_requests)

**Comment by J0WI** [comment]
You mixed various changes on unrelated hosts together in this PR.
Could you split this up in to multiple PRs please?

**Comment by galeksandrp** [comment]
#5164
#5165
#5166
#5167
#5168
#5169
#5170
#5171
#5172
#5173

**Pull request closed without merging**